public/images/aboutImage.tsx

```diff
@@ -0,0 +1,5 @@
+import Image from 'next/image'
```

Comment:
Components should not be placed inside the public folder. You can consider replacing it with a `about.jpg` image.